From geert@linux-m68k.org Tue Dec  8 09:38:03 2015
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Tue,  8 Dec 2015 18:37:39 +0100
Subject: [PATCH 3/5] clkdev: const-ify connection id to clk_add_alias()
To: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Cc: Simon Horman <horms@verge.net.au>, Magnus Damm <magnus.damm@gmail.com>, ltsi-dev@lists.linuxfoundation.org, Russell King <rmk+kernel@arm.linux.org.uk>, Geert Uytterhoeven <geert+renesas@glider.be>
Message-ID: <1449596261-31005-4-git-send-email-geert+renesas@glider.be>


From: Russell King <rmk+kernel@arm.linux.org.uk>

The connection id is only passed to clk_get() which is already const.
Const-ify this argument too.

Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
(cherry picked from commit b3d8d7e89fab374d731dfb46fe048f09766ca9c8)
Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
---
 drivers/clk/clkdev.c   |    6 +++---
 include/linux/clkdev.h |    2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/clk/clkdev.c
+++ b/drivers/clk/clkdev.c
@@ -288,10 +288,10 @@ clkdev_alloc(struct clk *clk, const char
 }
 EXPORT_SYMBOL(clkdev_alloc);
 
-int clk_add_alias(const char *alias, const char *alias_dev_name, char *id,
-	struct device *dev)
+int clk_add_alias(const char *alias, const char *alias_dev_name,
+	const char *con_id, struct device *dev)
 {
-	struct clk *r = clk_get(dev, id);
+	struct clk *r = clk_get(dev, con_id);
 	struct clk_lookup *l;
 
 	if (IS_ERR(r))
--- a/include/linux/clkdev.h
+++ b/include/linux/clkdev.h
@@ -39,7 +39,7 @@ void clkdev_add(struct clk_lookup *cl);
 void clkdev_drop(struct clk_lookup *cl);
 
 void clkdev_add_table(struct clk_lookup *, size_t);
-int clk_add_alias(const char *, const char *, char *, struct device *);
+int clk_add_alias(const char *, const char *, const char *, struct device *);
 
 int clk_register_clkdev(struct clk *, const char *, const char *, ...);
 int clk_register_clkdevs(struct clk *, struct clk_lookup *, size_t);
